### DISCUSSION TITLE
Anchor Links working in Development but not in Production

### DISCUSSION THREAD

**Discussion by nonoumasy** [vercel/next.js#35587] [Help]
I looked for solutions online but couldn't find an answer.

1. I have Link pointing to a dynamic route with #anchor links
2. I have a <a> tag. I even repeated the href path but that didn't work either.
3. sections with anchor links have an id for that title.
4. It works in development, ie it opens the page and goes to the section of the page with the id of that title

```    
...
    <Link href={`/story/${item.slug}/#${item.title}`} prefetch={false} passHref >
    <a target="_blank" href={`/story/${item.slug}/#${item.title}`}>
      {item?.image?.includes("youtube") ? (
        <figure>
          <img
            loading="lazy"
            src={`https://img.youtube.com/vi/${item.image
              ?.split("/")
              .pop()}/hqdefault.jpg`}
            className="timeline-image"
          />
        </figure>
      ) : (
        <figure
          onError={(e) => (e.target.src = "https://i.imgur.com/ARuzyWH.png")}
        >
          <img
            loading="lazy"
            src={item?.image ? item?.image : "https://i.imgur.com/ARuzyWH.png"}
            alt={item?.title}
            className="timeline-image"
          />
        </figure>
      )}
      <div style={{ fontSize: 10, color: "reset" }}>
        {item.eventYear} {monthNames[item.eventMonth - 1].slice(0, 3)}
      </div>
      <figcaption className="title" style={{ lineHeight: 1.3, fontWeight: 600 }}>
        {item?.title}
      </figcaption>
    </a>
  </Link>
  ...
```

**Comment by icyJoseph** [comment]
I don't think you need to `href` on the nested `a` tag.

Have you tried passing `scroll={false}` to the Link? I think otherwise, it tries to scroll to the top, if you just use an `a` tag without `Link`, just for testing, does it do the trick?

**Reply by nonoumasy** [reply]
Thanks @icyJoseph. That does work! I didn't know that was default behavior for the Link.
I also tried just using <a> but this didn't work. 

Update:
For some reason, its not working again.

It might have something to do with browser since when I copy and paste this in Safari and Fireofx, it scrolls to id :
https://history-maps.com/story/The-Crusader-States-1099-1047#Rise%20of%20the%20Artuqids

But in Chrome it doesn't work, so it might not be a NExtjs issue.
